app/models/post.rb

```diff
@@ -270,24 +271,19 @@ def set_last_user
     self.last_user = user
   end
 
-  def timestamp_attributes_for_update
-    # Makes Rails treat edited_at as a timestamp identical to updated_at
-    # if specific attributes are updated. Also uses tagged_at if there
-    # are no replies yet or if the status has changed.
-    # Be VERY CAREFUL editing this!
-    return super if skip_edited
-    return super + [:edited_at] if replies.exists? && !status_changed?
-    super + [:edited_at, :tagged_at]
+  # timestamps start existing between before_save and before_create/update
+  # TODO: ensure tested
```

Comment:
fix todo before merging

Reply:
still has a todo :D

Reply:
I think I'm confident that `Post should have the right timestamps` tests this, so I'll remove the TODO.